### PR TITLE
[Chore] e2e edge asymmetric cases

### PR DESCRIPTION
**Pull request by jonycfu**
**PR checklist**

- [X ] Ensure you have added or ran the appropriate tests for your PR.
- [ X] DCO signed

**What type of PR is this?**
Chore. build optimizations
**What this PR does / why we need it**:
E2E UI testing on abnormal and edge cases of datasets
**Which issue(s) this PR fixes**:
N/A
**Special notes for your reviewer**:
ALL tests pass now; 
Screen Shot 2022-08-04 at 10 51 55 AM

**Does this PR introduce a user-facing change?**:
No

NONE

### CODE REVIEW

File: static_report/src/sdlc/generate-e2e-data.sh [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
 # For generating comparison_data.json from `piperider-getting-started`
 # NOTE: This file assumes you are calling from the root of repo
+# NOTE: Pass parameters to shell call e.g. $1
+if [ $2 = "clean" ]; then
+	rm -rf piperider-getting-started
+fi
 
-# clone repo
-git clone https://github.com/InfuseAI/piperider-getting-started.git
+# First 2 run.jsons
```
Comment (neighborhood999) [inline static_report/src/sdlc/generate-e2e-data.sh:9]:
```suggestion
# First 2 run.json
```

Comment (jonycfu) [reply static_report/src/sdlc/generate-e2e-data.sh:9]: should be plural, since it's two jsons, not one :)

Comment (neighborhood999) [reply static_report/src/sdlc/generate-e2e-data.sh:9]: Sorry for my poor English.

**Review by neighborhood999 — APPROVED** [review]
